docs/readmes/orc8r/dev_minikube.md

```diff
@@ -105,6 +105,31 @@ helm template orc8r charts/secrets \
   kubectl apply -f -
 ```
 
+### Optional: fluentd secrets
+
+To use fluentd on the minikube deployment, create additional fluentd secrets
+
+```bash
+cd ${CERTS_DIR}
+openssl genrsa -out dp_fluentd.key 2048
+openssl req -new -key dp_fluentd.key -out dp_fluentd.csr -subj "/C=US/CN=dp-fluentd.$domain"
+openssl x509 -req -in dp_fluentd.csr -CA certifier.pem -CAkey certifier.key -CAcreateserial -out dp_fluentd.pem -days 3650 -sha256
```

Comment:
Created but not applied in lines 125ff. Why?

Reply:
Sorry, my bad. These are indeed not necessary, I removed them from the documentation.